src/docker/job_control.py

```diff
@@ -176,27 +192,22 @@ def run_job(job: str, s3client: client) -> int:
         [],
     )
 
-    job_output = f"> {job_type}.stdout.txt 2> {job_type}.stderr.txt"
+    job_stdout_filename = f"{job_type}.stdout.txt"
+    job_stderr_filename = f"{job_type}.stderr.txt"
```

Comment:
The `job_stdout_filename` and `job_stderr_filename` variables can be eliminated if you change line 210 to:
```
execute_command(command, f"{job_type}.stdout.txt", f"{job_type}.stderr.txt")
```
I try to eliminate variables that are only used once or move them just above where they are used.